src/pages/[locale]/[...path]/index.md.astro

```diff
@@ -0,0 +1,16 @@
+---
+import ToMarkdown from '@components/ToMarkdown/ToMarkdown.astro';
+import Page from './index.astro';
+export { getStaticPaths } from './_index';
+
+export const partial = true;
+
+Astro.response.headers.set('Content-Type', 'text/markdown; charset=utf-8');
```

Comment:
HTML entities in the content are still encoded (`<` becomes `&lt;` etc). This is not the cause of the `ToMarkdown` component as it also happens with the HTML comment below. So Astro forces this higher up I believe. Haven't been able to stop encoding or force decoding 🤷 .